data-access-layer/bosh/BoshDirectorClient.js

```diff
@@ -916,8 +915,9 @@ class BoshDirectorClient extends HttpClient {
           user: tempUser,
           public_key: publicKey
         }
-      }
-    }, 200, deploymentName);
+      },
```

Comment:
can add and use all http status codes from here - https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/c812e07c2e8332d1fbf8f3cce5053e8a9d1159e0/common/constants.js#L99